cmd/kots/cli/velero.go

```diff
@@ -160,6 +161,7 @@ func VeleroConfigureInternalCmd() *cobra.Command {
 	}
 
 	cmd.Flags().Bool("skip-validation", false, "skip the validation of the internal store endpoint/bucket")
+	cmd.Flags().Bool("with-minio", true, "when set, kots will deploy minio for NFS snapshot locations")
```

Comment:
Help message is copy-pasta from NFS. Should mention something about internal snapshots.